.travis.yml

```diff
@@ -0,0 +1,38 @@
+branches:
+  only:
+  - master
+  - /^v.*$/
```

Comment:
This script is set up to only try to deploy to pypi if you push a tag and that tag starts with "v" you can alter that pattern here.